test/createGuild.js

```diff
@@ -24,10 +24,10 @@ client.on('ready', async () => {
     assert.strictEqual(guild.afkTimeout, 60);
     assert.strictEqual(guild.systemChannel.name, 'system-channel');
     await guild.delete();
-    client.destroy();
   } catch (error) {
     console.error(error);
-  }
+  } finally {
+    client.destroy();
```

Comment:
Missing a closing bracket.
```suggestion
    client.destroy();
  }
```

Reply:
thanks